src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1223,6 +1227,41 @@ private function registerPropertyAccessConfiguration(array $config, ContainerBui
         ;
     }
 
+    private function registerAmqpConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader)
+    {
+        $loader->load('amqp.xml');
+
+        $defaultConnectionName = $config['default_connection'] ?? null;
+
+        $match = false;
+        foreach ($config['connections'] as $name => $connection) {
+            $container
+                ->register("amqp.broker.$name", Broker::class)
+                ->setFactory(array(Broker::class, 'createWithDsn'))
+                ->addArgument($connection['dsn'])
+                ->addArgument($connection['queues'])
+                ->addArgument($connection['exchanges'])
+                ->setPublic(false)
```

Comment:
already the default

Reply:
https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php => there are 14 times `->setPublic(false)`. Looks like there is an "easy first contribution" here